src/internal/connector/onedrive/item.go

```diff
@@ -297,15 +297,32 @@ func filterUserPermissions(ctx context.Context, perms []models.Permissionable) [
 // and kiota drops any SetSize update.
 // TODO: Update drive name during Issue #2071
 func sharePointItemInfo(di models.DriveItemable, itemSize int64) *details.SharePointInfo {
-	var id, driveName, driveID, weburl string
+	var driveName, siteid, driveID, weburl, email string
```

Comment:
* email -> creatorEmail for clarity
* siteid -> siteID for standard golang naming convention.

```suggestion
	var driveName, siteid, driveID, weburl, creatorEmail string
```